Sources/MapboxCoreNavigation/PredictiveCacheLocationOptions.swift

```diff
@@ -0,0 +1,38 @@
+
+import Foundation
+import MapboxNavigationNative
+
+/// `PredictiveCacheLocationOptions` controls various configurations for a `Predictive Caching` mechanic.
+public class PredictiveCacheLocationOptions {
+    
+    /// How far around the user's location we're going to cache, _in meters_.
+    ///
+    /// Defaults to 2 km
+    public var currentLocationRadiusInMeters: UInt32
+    /// How far around the active route we're going to cache, _in meters_ (if route is set).
+    ///
+    /// Defaults to 500 m
+    public var routeBufferRadiusInMeters: UInt32
+    /// How far around the destination location we're going to cache, _in meters_ (if route is set).
+    ///
+    /// Defaults to 5 km
+    public var destinationLocationRadiusInMeters: UInt32
```

Comment:
Use `CLLocationDistance` (which is measured in meters) for any distance measurement in the Swift codebase.